### ISSUE TITLE
Dataset size and Lambda

### ISSUE THREAD

**Issue by MatheusMRFM** [williamd4112/tensorflow-action-conditional-video-prediction#1]
Hello there! First of all, congratulations on your implementation!

I have a few questions regarding your training process:

1. What was the size of your dataset (how many frames)?
2. What value did you use for the regularization parameter (lambda)?
3. Do you have a plot of of the loss function over the course of training? If not, do you remember with what value (approximately) the loss function started and to what value it converged?
4. How long did it take for you to train your model?

I'm sorry for many asking questions, but I have very limited computational resources, so I can't just go experimenting different parameters.

Thank you!
